### ISSUE TITLE
Edit notes for repeated tasks

### ISSUE THREAD

**Issue by KatrinIhler** [johannesjo/super-productivity#2938]
I'm honestly not sure how this could best be achieved, and I imagine it could be quite complicated, but I'll just describe my use case here:

I use the description of scheduled tasks to take notes of things I want to talk about for upcoming meetings. This is quite handy for me, as the notes aren't kept separately, but are directly related to the task which will magically appear in my Today view at the correct time.

A lot of my meetings are on a regular schedule, e.g. once a week. I would love to use repeating tasks for this instead of scheduling each of them separately as needed, but I can't edit the notes of a repeating task until the time it is due. I guess I could use the default notes for this, but then the notes will appear on all tasks from then on, unless I clear them.

It would be incredibly handy to see the next upcoming repeated task somewhere in the Scheduled view and be able to edit its notes before that task is due. But I guess this would require some changes regarding the way repeating tasks are initialized...

**Comment by KatrinIhler** [comment]
And I'm done spamming your issues now btw. ;) I've been using your app for work for quite a while now and I really appreciate it - couldn't stick with any other software solution for long until I found this one. Some of my colleagues have also been using it.  Just the right amount of complexity and great features if you do software engineering work and have to do time tracking. Hope this will exist for a long time. 

Might contribute something small in the future if I find the time (and figure out how this thing works, my experience with frontend stuff is limited), but at least now I finally went through my notes with bugs and feature requests... which I kept in the description of a task, of course. ;)

**Comment by Jagdfalke** [comment]
Maybe a similar concept to the current calendar integration would help solve this issue? With the new calendar integration you can now add future events from your timeline to your inbox. It the same concept would apply to repeated tasks, you could add future instances from the timeline and add your notes.

@KatrinIhler would that solve your issue?
@johannesjo is that feasible?

**Comment by johannesjo** [comment]
> @johannesjo is that feasible?

Yes! This should be possible.

**Comment by KatrinIhler** [comment]
There is a calendar integration?? Somehow this slipped by me... That sounds so useful! Gonna give it a spin soon :)

**Comment by Jagdfalke** [comment]
@KatrinIhler Thanks for your feedback!

Regarding the calendar integration: You are correct, authentication is not supported. Are you sure you need it though? You might be able to create a public link with minimal details. That is all SP needs to pull the tasks.

**Comment by KatrinIhler** [comment]
> Are you sure you need it though? You might be able to create a public link with minimal details. That is all SP needs to pull the tasks.

We are using nextcloud, and there are multiple possibilities to generate public links, but so far none of them have worked for me (either errors or just no events synced). Is there some documentation for what kind of format SP expects the link to be in?

Also feel free to move this discussion somewhere else, I don't want to spam this issue.

**Comment by frainz-de** [comment]
Yup, I'm watching the issue and am looking forward to the feature :)

**Comment by KatrinIhler** [comment]
Not really relevant to me anymore, I've made the calendar integration work with a combination of a nextcloud API key and putting those credentials into the URL the calendar is pulled from. Not ideal, but I like it better than a public subscription link. So I don't really have a need for this use case anymore.

**Comment by GlassedSilver** [comment]
Am I dreaming? Are repeat tasks really set in stone until they come up and all edits to the task ONLY affect the one occurence, not future ones?

I'm trying to add sub-tasks, add attachments (links in my case) and well, brush up the notes. I can't quite understand if this is really missing or I'm just REALLY misinterpreting what this issue asked for and there is a way, but I just didn't tickle the right button.

If someone could fill me in if I'm tripping or the issue may need to be looked at again I'd be very grateful, because I could swear I edited a repeating task before, but then again maybe that edit just went into the one occurence and didn't propagate to the future ones and I keep missing something about one of my tasks.

**Comment by johannesjo** [comment]
@GlassedSilver this is not the rigth place for your question. Please either create a new issue or post your question here:
https://github.com/super-productivity/super-productivity/discussions

**Comment by GlassedSilver** [comment]
> [@GlassedSilver](https://github.com/GlassedSilver) this is not the rigth place for your question. Please either create a new issue or post your question here: https://github.com/super-productivity/super-productivity/discussions

Sorry, I found this issue due to Google and it seemed closely related to the behaviour I noticed, that's why I posted here. In general I think it's wise to attach related questions to the issue that already exists.

I'll gladly create a new issue however when I get to it, thanks for your suggestion. 👍

**Comment by johannesjo** [comment]
Thank you! :)
